bokeh/sphinxext/bokeh_roles.py

```diff
@@ -13,8 +13,12 @@
 
```

Comment:
Line 9 and 10 should be updated (number and typos):

> This module provides six additional roles that can be used to easily link to various
> resources in the Bokeh Github repository:

Reply:
@tcmetzger Can you make as suggestion here? then I can commit everything as one batch from the UI (Just FYI in case: it's possible to highlight multiple lines for multi-line suggestions)

Reply:
I tried, but the line I'd like to change seems to be too far away from the lines you changed, so I can't select them (I don't get the little "+" icon on any line above line 13).

Reply:
ah that is annoying

Reply:
I left out the  number think it's hopefully future-proof now!